src/app/components/mayastor/mayastor.component.ts

```diff
@@ -17,6 +19,27 @@ export class MayastorComponent implements OnInit {
   }
 
   ngOnInit(): void {
+    console.log('maystor component :)');
+  }
+
+  getStatus(tests) {
+    try {
+      if (tests) {
+        const failed = tests.filter(test => test.status === 'FAILED').length;
+        if (failed !== 0) {
+          return 'FAILED';
+        } else {
+          return 'PASSED';
+        }
+      }
+    } catch (error) {
+      return '_';
```

Comment:
why this ? what does _ mean here.

Reply:
usually if any required field <or> index empty, then the conditions/function break happened. At that time directive-loop in HTML will break and won't go for the next rows/elements.
here in above function, if anyThing went wrong it will print the " _ " in the dashboard without breaking the iteration.

noticed: tryCatch method not require for status function . Removed from code